src/css-rule-set-to-string.js

```diff
@@ -5,9 +5,14 @@ import camelCasePropsToDashCase from './camel-case-props-to-dash-case';
 import mapObject from './map-object';
 import {getPrefixedStyle} from './prefixer';
 
+function quoteValueIfNeeded(property, value) {
```

Comment:
Your function definition should end up looking something like this to enable type checking with flow:

```js
function quoteValueIfNeeded(propertyName: string, value: any): string {

}
```